client/dashboard/sites/settings-repositories/index.tsx

```diff
@@ -114,42 +115,67 @@ function SiteRepositories() {
 	const navigate = useNavigate( { from: '/sites/$siteSlug/settings/repositories' } );
 	const canConnect = hasHostingFeature( site, HostingFeatures.DEPLOYMENT );
 
+	const [ showBackToDeployments, setShowBackToDeployments ] = useState( true );
+
```

Comment:
Nit: I think we can remove the state var here, because navigating away from this page unmounts the snackbar component anyways.